pkg/plugin/datasource.go

```diff
@@ -55,8 +55,8 @@ var (
 )
 
 // NewTwinMakerDatasource creates a new datasource instance.
-func NewTwinMakerDatasource(settings models.TwinMakerDataSourceSetting) *TwinMakerDatasource {
-	c, err := twinmaker.NewTwinMakerClient(settings)
+func NewTwinMakerDatasource(settings models.TwinMakerDataSourceSetting, ctx context.Context, ) *TwinMakerDatasource {
```

Comment:
minor nit (here and below): `ctx` is by custom usually the first argument, for readability / visibility

Reply:
Done